package.json

```diff
@@ -13,7 +13,7 @@
     "build_test": "rm -rf dist/ && npm run build_es6 && npm run build_cjs && jasmine",
     "build_docs": "./docgen.sh",
     "test": "jasmine",
-    "perf": "protractor protractor.conf.js",
+    "perf": "webdriver-manager update && protractor protractor.conf.js",
```

Comment:
What about put `webdriver-manager update` into `build_perf` instead? that'll allow to run perf test without calling it each time.